src/main/java/com/powsybl/openloadflow/network/impl/LfAreaImpl.java

```diff
@@ -7,17 +7,19 @@
  */
 package com.powsybl.openloadflow.network.impl;
 
+import com.powsybl.commons.PowsyblException;
 import com.powsybl.iidm.network.*;
 import com.powsybl.iidm.network.util.SV;
 import com.powsybl.openloadflow.network.*;
+import com.powsybl.openloadflow.util.Evaluable;
 import com.powsybl.openloadflow.util.PerUnit;
 
 import java.util.*;
 
 /**
  * @author Valentin Mouradian {@literal <valentin.mouradian at artelys.com>}
  */
-public class LfAreaImpl extends AbstractPropertyBag implements LfArea {
+public class LfAreaImpl extends AbstractPropertyBag implements LfArea, PropertyBag {
```

Comment:
same, would be cleaner to extend AbstractElement properly so you won't have to do all these overrides / exceptions